Makefile

```diff
@@ -93,23 +94,23 @@ coverhtml: coverage ## Generate HTML coverage report
 	@go tool cover -html=coverage_cleaned.cov -o coverage_unit.html
 
 .PHONY: e2e-test-docker-image ## Build Docker image that is pushed to a registry and used in e2e tests
-e2e-test-docker-image: deploy/e2e-test-image/Dockerfile
-	@docker build -t $(E2E_TEST_DOCKER_IMAGE) deploy/e2e-test-image
+e2e-test-docker-image: deploy/dockerfiles/${E2E_TEST_DOCKER_CONTAINER}/$(E2E_TEST_DOCKER_TAG)/Dockerfile
```

Comment:
Should we add a rule (such as git clone) for `deploy/dockerfiles/${E2E_TEST_DOCKER_CONTAINER}/$(E2E_TEST_DOCKER_TAG)/Dockerfile:` to make this Makefile work even outside the CI/CD?

Reply:
Great :blush: